python/cuml/common/array_sparse.py

```diff
@@ -135,6 +135,7 @@ def to_output(self, output_type='cupy',
 
             - 'cupy' - to cupy array
             - 'scipy' - to scipy (host) array
+            - 'numpy' - to scipy (host) array
```

Comment:
Does scipy imply sparse and numpy dense? Maybe confusing to have both? Possibly I'm missing the motivation

Reply:
This was more of a safeguard, since the cuml array doesn't know about Scipy, I am using `numpy` as a surrogate for "I want my results on host rather than device. I recall there being an explicit error that prompted me to do this but I can't remember exactly what it was off the top of my head. I could go either way, really, but I do recall it had something to do w/ the recognized output types. Ideally, the current output type resolution would understand the 'scipy' option (and even more ideally, we would use enums instead of strings).

Reply:
I think the strings, for all their faults, are probably still more pythonic... I'm ok keeping numpy as an option but just updating the comment to be clear whether these are sparse or dense arrays coming out